py3status/modules/dropboxd_status.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-Display dropboxd status.
+Display service status of Dropbox daemon.
```

Comment:
Does this make sense? `Display service status for Dropbox` maybe? 
EDIT: No... We want Dropbox daemon, not the service itself. Will leave it alone, mate.